### DISCUSSION TITLE
UI/UX optimization

### DISCUSSION THREAD

**Discussion by d4f5409d** [osmandapp/OsmAnd#12489] [Ideas]
It would be better to make the UI more rounded, somewhere bigger. For instance, the navigation panel should be more rounded, and there should be more space between stuff.

**Comment by vshcherb** [comment]
It's possible it's a matter of taste (round / rectangular) + Look and feel or there is something related to usability there?

**Reply by d4f5409d** [reply]
Kinda taste and kinda usability. I mean it would be good to take inspiration in design from Google Maps, Mapy.cz and Organic Maps.

**Reply by Lee-Carre** [reply]
@d4f5409d

> UI more rounded

> Kinda taste and kinda usability.

How is rounding corners anything to do with *usability* (interaction-design)?(!) Suggests that you don't know what usability *is*.

Seems to be entirely a matter of aesthetic taste, possibly visual design (though I'm mindful of how much visual style seems to be fashions & trends, rather than based in anything of substance).

When development time is limited, prettiness comes much further down the priority list than functionality.

I even coined a term for this, years ago; pink phone syndrome. Caring only about the superficial (e.g. colour of the case, which is easy to change (re-paint)), regardless of if the internals are any good for serving its primary purpose. Those afflicted with PPS tend to end up with crap in a shiny wrapper, a gold-plated dog-turd. The alarming part is that they incentivise the market to cater to such shallowness (which is probably behind all the app-killing battery-saving changes we're seeing; people whining that their tiny 3Ah doesn't last all week when they're using it most of the day (with the display brightness perma-set to max), yet scoffing at the mere suggestion of using an external battery (because it would cramp their style, or clash with their outfit)).

Anyway. Rounded corners risk decreasing usability; they'll have a performance hit, reducing responsiveness. They also, inherently, reduce the count of usable pixels (or display area, whichever you're measuring), which means that either fewer elements (or information, like the actual map) can be displayed, or those which are displayed are more ‘cluttered’.

> take inspiration in design from Google Maps

You're seriously suggesting this on a project which inherently rejects [Google](//www.stallman.org/google.html)'s proprietary ways, and makes a point of preserving freedom & respect for users? Oh dear.

I must wonder why, if Google is so much prettier, why you don't simply volunteer for that (equivalent of a) comfortable prison.

**Reply by Lee-Carre** [reply]
> [is there] something related to usability there?

No. To the extent that I think the title is misleading & should be changed.

**Comment by d4f5409d** [comment]
I checked the iOS version and OMG it's so good! Please make something similar to the Android version in terms of UI!
